### PR TITLE
Instance rec documentation

### PR DESCRIPTION
**Pull request by SamDesmondKing**

### CODE REVIEW

File: pages/user guides/instance_recommendation.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## How does it work?
+
+Tidal Accelerator's cloud instance recommendation uses a proven scientific approach to compare the CPU performance of your on-premise servers with their cloud compute equivalent. This allows us to recommend instances which will meet performance requirements for your workloads while delivering the best possible value, ensuring that workloads are not over-provisioned. 
+
+We consider CPU age and processing power, as well as RAM and storage usage to generate our recommendations. As you discover more about your server infrastructure, your instance recommendations change and become more refined, based on real-world usage data. 
+
+Under the hood, we leverage the approach described in Amazon Web Service's whitepaper <a target="_blank" href="https://s3.ca-central-1.amazonaws.com/tidal.assets/aws-demystifying-vcpus.pdf">Demystifying the Number of vCPUs for Optimal Workload Performance</a>.
+
+This scientifically proven methodology drives the CPU performance aspect of our recommendations.
```
Comment (justinbarclay) [inline pages/user guides/instance_recommendation.md:24]:
```suggestion
This proven methodology drives the CPU performance aspect of our recommendations.
```

File: pages/user guides/instance_recommendation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+{% include image.html file="instance_rec_1.png" %}
+
+To assign servers to a project, you must create a migration wave. Simply create the migration wave for your project.
+
+{% include image.html file="instance_rec_2.png" %}
+
+After navigating to your migration wave, you can then add servers (and their dependencies, if required).
+
+All servers in this project will use the project's cloud provider and cloud region by default. If you need more granular control over cloud region, you can set this at the server level as well. Be aware that changing cloud provider or region at the project level will clear server-level cloud region settings, and re-generate recommendations for all servers in the project.
```
Comment (justinbarclay) [inline pages/user guides/instance_recommendation.md:40]: This should be in a warning block
> Be aware that changing cloud provider or region at the project level will clear server-level cloud region settings, and re-generate recommendations for all servers in the project. 
Comment

File: pages/user guides/instance_recommendation.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+{% include image.html file="instance_rec_2.png" %}
+
+After navigating to your migration wave, you can then add servers (and their dependencies, if required).
+
+All servers in this project will use the project's cloud provider and cloud region by default. If you need more granular control over cloud region, you can set this at the server level as well. Be aware that changing cloud provider or region at the project level will clear server-level cloud region settings, and re-generate recommendations for all servers in the project. 
+
+### Gather data
+
+After servers are assigned to a project, instance recommendation occurs automatically when your servers have enough data for us to begin making recommendations.
+
```
Comment (justinbarclay) [inline pages/user guides/instance_recommendation.md:45]: There should be an info block here saying some thing like "The recommendation process sometimes can take several minutes to complete."

File: pages/user guides/instance_recommendation.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- Storage used
+- Number of logical processors
+
+See [our documentation](machine_stats.html) for detailed instructions on gathering machine stats with Tidal Tools.
+
+From here, we can begin generating recommendations for this server. You can find your recommendation in the server overview under Transition Planning > Cloud Instance Target. If server doesn't have a recommendation yet, you should be able to hover over this field and see what information you need to gather to generate the recommendation.
+
+{% include image.html file="instance_rec_3.png" %}
+
+### Refine your recommendation
```
Comment (justinbarclay) [inline pages/user guides/instance_recommendation.md:58]:
```suggestion
### Refining your recommendation
```

**Review by justinbarclay — APPROVED** [review]
@SamDesmondKing this is looking really good

**Comment by justinbarclay** [comment]
nit: for consistency can we switch to tidal blue for highlights?
image

File: pages/user guides/instance_recommendation.md (hunk 1 above)
Comment (SamDesmondKing) [reply pages/user guides/instance_recommendation.md:24]: I'm using the language from the whitepaper here but happy to remove this if it doesn't sound right
image

**Comment by SamDesmondKing** [comment]
> nit: for consistency can we switch to tidal blue for highlights? (I'm also interested to find out if we use yellow as a highlight in other parts of the guides)

Happy to change it, however we do have a lot of blue in our UI so I'm concerned it won't appear 'highlighted' enough. I went with tidal orange 2 because I saw it used [here](https://deploy-preview-273--tidal-guides.netlify.app/assessment-projects#creating-a-project) and [here](https://deploy-preview-273--tidal-guides.netlify.app/azure-migrate#review).

We don't seem to have a standard approach to doing this, from what I can tell it breaks down like this:

Blue
https://deploy-preview-273--tidal-guides.netlify.app/public-urls
https://deploy-preview-273--tidal-guides.netlify.app/ansible-tower
https://deploy-preview-273--tidal-guides.netlify.app/postman-docs
https://deploy-preview-273--tidal-guides.netlify.app/troubleshoot-ansible

Red
https://deploy-preview-273--tidal-guides.netlify.app/azure-migrate
https://deploy-preview-273--tidal-guides.netlify.app/integrate-with-jira

Tidal orange
https://deploy-preview-273--tidal-guides.netlify.app/assessment-projects#creating-a-project
https://deploy-preview-273--tidal-guides.netlify.app/azure-migrate#review

I would prefer to use red or orange over blue but again I'm happy to change it because blue seems to be what we do the most.

**Comment by SamDesmondKing** [comment]
This is ready for another look :)

**Comment by justinbarclay** [comment]
Cool, if you found it used elsewhere all good then :), I'm happy to use a colour that's already being used

File: pages/user guides/instance_recommendation.md (hunk 1 above)
Comment (justinbarclay) [reply pages/user guides/instance_recommendation.md]: Great, if you're quoting them, that's fine. I _disagree_ with their assertion it's scientific, but as long as we aren't making that claim it's all good :)

**Pull request merged**